README.fr.md

```diff
@@ -0,0 +1,42 @@
+## Liste noire de serveurs
+
+Ce dépôt contient une `blacklist.txt` (liste noire) de domaines XMPP qui sont
+utilisés par des spammeurs et qui ne répondent pas aux plaintes d'abus. Ces
+serveurs sont ajoutés et retirés en suivant les règles listées ci-dessous. Les
+changements apportés à cette liste sont documentés dans chaque contribution.
+
+### Ajouter un serveur à la liste noire
+
+Un serveur est ajouté à la liste noire après avoir suivi les étapes ci-dessous,
+si aucune n'a abouti à réduire le trafic de spam du serveur:
```

Comment:
Always use a narrow non-breakable space (U+202F, “ ”) before colons and other double punctuation marks in French, here and below.